templates/server/puppet.conf.erb

```diff
@@ -15,10 +15,7 @@
 <% end -%>
     strict_variables = <%= scope.lookupvar("puppet::server_strict_variables") %>
 
-<% if scope.lookupvar("puppet::server_directory_environments") -%>
-    environmentpath  = <%= scope.lookupvar("puppet::server_envs_dir") %>
-    basemodulepath   = <%= [scope.lookupvar("puppet::server_common_modules_path")].flatten.join(':') %>
-<% elsif scope.lookupvar("puppet::server_git_repo") ||
+<% if scope.lookupvar("puppet::server_git_repo") ||
```

Comment:
These conditionals aren't quite right now for a couple of cases.. it used to work thanks to the "elsif":

1. When both server_directory_environments & server_git_repo are true, it used to only add environmentpath/basemodulepath.  I think now it will add env/basemodulepath into `[main]` and also add manifest/modulepath in `[master]` which isn't right.
2. ditto with server_dynamic_environments

I think this if statement needs to be: `if !server_directory_environments && (server_git_repo || server_dynamic_environments)` (pseudo-code)